server/schedule/scheduler.go

```diff
@@ -45,6 +45,7 @@ type Cluster interface {
 	RegionReadStats() []*core.RegionStat
 
 	// get config methods
+	GetOpt() NamespaceOptions
```

Comment:
`GetNamespaceOpt`?

Reply:
cause here means return the opt field of itself, I think GetOpt() is clear